src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunBWAMEMViaCommandLineTest.java

```diff
@@ -0,0 +1,116 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Tests RunBWAMEMViaCommandLine.
+ * To make this test runnable, "-DPATHTOBWA" should be set properly.
+ */
+public final class RunBWAMEMViaCommandLineTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "spark/sv/RunBWAMEMViaCommandLine");
+    private static final File expectedSam = new File(TEST_DATA_DIR, "expected.sam");
+    private static final Path ref = Paths.get(b37_reference_20_21);
+    private static final Path bwaPath = getBwaPath();
+    private static Path getBwaPath(){
+        String s = System.getProperty("PATHTOBWA");
+        if(null==s){
+            s = getNonNullEnvironmentVariable("PATHTOBWA");
+            if(null==s) {
```

Comment:
\[FindBugs\] INFO: RCN: Redundant nullcheck of s, which is known to be non\-null in org\.broadinstitute\.hellbender\.tools\.spark\.sv\.RunBWAMEMViaCommandLineTest\.getBwaPath\(\)